File: solved.ac/implementation/javascript/2012.js
const fs = require("fs")
let inputs = fs.readFileSync("./dev/stdin").toString().trim().split("\n")
const n = +inputs.shift()
inputs = inputs.map(Number).sort((a, b) => a - b)

let result = 0

for (let i = 0; i < inputs.length; i++) {
  result += Math.abs(i + 1 - inputs[i])
}

console.log(result)

File: solved.ac/python/2012.py
n = int(input())
expected = []
for _ in range(n):
    expected.append(int(input()))

# 정렬
expected.sort()

# 불만족도 합
result = 0
for i in range(1, n+1):
    result += abs(i-expected[i-1])
print(result)